CHANGELOG.md

```diff
@@ -7,13 +7,20 @@ accidentally triggering the load of a previous DB version.**
 
 ## 1.3.1 (unreleased)
 
+This release contains bugfixes.
```

Comment:
Open question -- should we have something a bit more descriptive here? On some release platforms like Ubuntu/Debian, I have to fill out a bullet point changelog, and usually I put whatever is here as those bullets. I guess I could use just the things listed under Bugfixes

Reply:
Yes.  I think one particular relevance is that whether any are security vulnerabilities (I think not), stability bug fixes (maybe just for PG11 given VACUUM?), etc.  In other words, it would be useful if this header could give user a sense of how important would it be for them to upgrade?

Reply:
Sounds like a great idea